doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_memory.rst

```diff
@@ -232,54 +195,54 @@ Access control
 Fast global RAM (RAM21)
 -----------------------
 
-.. image:: images/nrf54h20_memory_map_ram21.svg
-   :width: 300 px
-
 The Fast global RAM is a part of the global RAM tightly coupled with the Fast Lightweight Processor.
 Access to this memory block from the FLPR and fast peripherals' DMA (I3C, CAN, PWM120, UARTE120, SPIS120, SPIM120, SPIM121) has minimal latency and can be performed without powering up any other parts of the system.
 Access to this memory from the local domains has higher latency than access to the general-purpose shared RAM.
 
 This memory is intended to store the code executed in the FLPR, the FLPR's data, the shared memory used for Inter-Processor Communication between the FLPR and the core managing the FLPR, and DMA buffers for the fast peripherals.
 
 Address range
-   0x2F890000 - 0x2F898000
+   0x2F890000 - 0x2F8A0000
 
 Size
-   32 KB
+   64 KB
 
 Access control
-   The FLPR and its owner have access to all partitions assigned to FLPR and its Inter-Processor Communication.
-   Each of the memory partition assigned for DMA of the fast peripherals is accessible from the core owning given set of the peripherals.
+   The FLPR and its owner have access to all partitions assigned to FLPR and its Interprocessor Communication.
+   Each one of the memory partitions assigned for the DMA of the fast peripherals is accessible from the core owning the given set of peripherals.
    The FLPR and the fast peripherals are by default owned by the Application Core.
    This ownership and matching memory access rights can be customized in UICRs.
```

Comment:
I think these lines should be removed. The remaining lines describe the access rules well enough.
```suggestion
```